README.md

```diff
@@ -92,6 +92,21 @@ export default Ember.Object.extend({
 });
 ```
 
+If you want to use i18n from within an initializer you need to make sure that it runs after the
+`ember-i18n` initializer has been executed.
+
+```js
+// app/instance-initializers/my-initializer.js
+
+export default {
+  name: 'my-initializer',
+  after: 'ember-i18n',
+  initialize: function({ container }) {
+    let i18n = container.lookup('service:i18n');
+    i18n.t('some.translation');
```

Comment:
I like this documentation block, but it might help to flesh out the use-case just a tiny bit:
```js
const i18n = container.lookup('service:i18n');
document.title = i18n.t('document.title');
```